### PR TITLE
adding pocType and jsonConfig field to certifyApi request

### PR DESCRIPTION
**Pull request by lalottacontinua**
and validation for expected json if tenantType == BMG

### CODE REVIEW

File: src/main/scala/com/ubirch/services/util/Validator.scala [hunk 1]
```diff
@@ -8,24 +8,43 @@ import com.ubirch.models.tenant._
...
+        case (_, Success(jValue))   => Some(jValue).validNel
       }
+
+  }
+
+  private def validateJsonWithSealId(header: String, str: String, jValue: JValue): AllErrorsOr[Option[JValue]] = {
+    Try(Serialization.read[SealJson](str)) match {
+      case Success(_) => Some(jValue).validNel
+      case Failure(error) =>
+        println(error)
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/services/util/Validator.scala:45]: forget to delete it?

**Review by Hiroki6 — APPROVED** [review]

**Review by Carlos-Augusto — APPROVED** [review]

File: src/main/scala/com/ubirch/services/poc/InformationProvider.scala [hunk 2]
```diff
@@ -28,12 +28,14 @@ trait InformationProvider {
...
   name: String,
+  pocType: String,
   endpoint: String,
   uuid: String,
   password: String,
   role: Option[String],
   location: Option[String],
-  cert: Option[String])
+  cert: Option[String],
+  config: Option[String])
```
Comment (Carlos-Augusto) [inline src/main/scala/com/ubirch/services/poc/InformationProvider.scala:38]: Just to get it: why is config a string?

Comment (lalottacontinua) [reply src/main/scala/com/ubirch/services/poc/InformationProvider.scala:38]: what would you like it to be?

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/services/poc/InformationProvider.scala:38]: I have no preference, I was just wondering why you decided to have it as a string.

Comment (lalottacontinua) [reply src/main/scala/com/ubirch/services/poc/InformationProvider.scala:38]: because I'm tired and didn't think of that I just could use the jvalue.

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/services/poc/InformationProvider.scala]: I am thinking if it give issues when parsing ... on the certify api. The type I selected was case class Config(sealId: UUID).

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/services/poc/InformationProvider.scala]: Will finish writing a test first...

**Pull request merged**